pyega3/libs/pretty_printing.py

```diff
@@ -26,12 +30,22 @@ def pretty_print_files_in_dataset(files):
         }
 
     """
-    format_string = "{:15} {:6} {:12} {:36} {}"
 
-    logging.info(format_string.format("File ID", "Status", "Bytes", "Check sum", "File name"))
-    for file in files:
-        logging.info(format_string.format(file.id, status_ok(file.status), str(file.size),
-                                          file.unencrypted_checksum, file.display_name))
-
-    logging.info('-' * 80)
-    logging.info("Total dataset size = %.2f GB " % (sum(file.size for file in files) / (1024 * 1024 * 1024.0)))
+    if as_json:
+        logging.info(json.dumps([{
+            "id": file.id,
+            "status": file.status,
+            "bytes": file.size,
+            "checksum": file.unencrypted_checksum,
+            "name": file.display_name
```

Comment:
I think the person who created the [original issue](https://github.com/EGA-archive/ega-download-client/issues/95) wanted to receive a few more fields, too. Here's his original wish:
```
 [
    {
        "fileId": "EGAF00000776139",
        "datasetId": "EGAD00001001261",
        "displayFileName": "D1UEHACXX_8_0.unaligned.bam",
        "displayFilePath": "EGAF00000776139.bam.cip",
        "fileName": "EGAF00000776139.bam.cip",
        "fileSize": 32007850278,
        "unencryptedChecksum": "d0e52dc9b681c4b575dcef5898c7edf8",
        "unencryptedChecksumType": "MD5",
        "fileStatus": "available"
    },
]
```

Reply:
@bela-juhasz @jorizci The user gave that as an example in his issue, but I think that he was just showing us what the response from the REST API looks like, it’s not clear that he actually needs all of those fields exactly. I think it would be better if we just expose the data we already expose in the output just in a new format. If there are specific fields he wants us to add to the output then we could explore that in a new issue, and maybe add them to both the JSON and table outputs at the same time.